doc/source/examples/extended_examples/Krylov/krylov_example.rst

```diff
@@ -42,16 +48,14 @@ and output impedance on the other end.
     mapdl.clear()
     mm = mapdl.math
 
+  
+Define parameters
+~~~~~~~~~~~~~~~~~
 
-    
-
-Parameters definition
-~~~~~~~~~~~~~~~~~~~~~
-
-This section defines some geometry parameters and analysis settings.
-As mentioned earlier, the geometry is a cylinder defined by its radius (``cyl_r``) and its length (``cyl_L``).
-The length of the duct is such that three complete wavelengths (``no_wl``) can fit in its length and can have
-ten elements per wavelength.
+Define some geometry parameters and analysis settings. As mentioned earlier, the geometry
+is a cylinder defined by its radius (``cyl_r``) and its length (``cyl_L``). The length
+of the duct is such that three complete wavelengths (``no_wl``) can fit in its length
+and can have ten elements per wavelength.
```

Comment:
What can have ten elements per wavelength. It seems that this needs to have a subject and be presented as a separate section.

Reply:
I would prefer to retain the edit that you have done which has this under the current section.